src/ui/windows/TogglDesktop/TogglDesktop/ui/Resources/TimelineConstants.cs

```diff
@@ -14,6 +14,8 @@ public static class TimelineConstants
         public const double GapBetweenOverlappingTEs = 5;
         public const double AcceptableBlocksOverlap = 1e-5;
         public const double MinGapTimeEntryHeight = 10;
+        public const int DefaultTimeEntryLengthInSeconds= 3600;

```

Comment:
`DefaultTimeEntryLengthInSeconds` should depend on `DefaultTimeEntryLengthInHours`

Reply:
Fixed